src/api/URLs.ts

```diff
@@ -43,7 +43,7 @@ class URLs {
   }
 
   public static cases(
-    filter: CaseSnoozeFilter,
+    filter: SnoozeState,
```

Comment:
Should this be an array of SnoozeState? In the ideal end state, that is (right now that'd break on the API side, obv)

Reply:
Right now we have:
- `active`
  - cases without a snooze
  - case with a snooze and has a snooze end value 
- `snoozed`
  - cases with a snooze
- `alarmed`
  - case with a snooze and has a snooze end value 

An array would let us continue to have the `active` state in the front end while also being able to get cases without a snooze independently.